[torch] Excluding test_fft_input_modification without MKL

letterx opened the pull request:
Summary: If torch compiled without MKL, this test fails with torch.fft requiring MKL support

Test Plan: CI

Differential Revision: D21051362

malfet commented on test/test_torch.py:
```diff
@@ -12063,6 +12063,7 @@ def _test(sizes, n_fft, hop_length=None, win_length=None, win_sizes=None,
         _test((10,), 5, 4, win_sizes=(1, 1), expected_error=RuntimeError)
 
     @skipIfRocm
+    @unittest.skipIf(not TEST_MKL, "PyTorch is built without MKL support")
```
```suggestion
    @skipCPUIfNoMkl()
```

afixOculus committed:
[torch] Excluding test_fft_input_modification without MKL (#36680)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/36680

If torch compiled without MKL, this test fails with torch.fft requiring MKL support

Test Plan: CI

Reviewed By: malfet

Differential Revision: D21051362

fbshipit-source-id: eda21814befcbbda5fa34cae59e1b6e771f952c2
